crates/codegen/parser/runtime/src/lib.rs

```diff
@@ -9,7 +9,16 @@ pub mod kinds;
 pub(crate) mod lexer;
 pub mod parse_error;
 pub mod parse_output;
+pub mod query_engine;
```

Comment:
Thouhgts about keeping the new (4) modules under a parent modue `query/mod.rs`?

Reply:
I was in two minds about this. So I guess that's 1.5 vs 0.5, so will do.